contrib/babelfishpg_tsql/src/dbcmds.c

```diff
@@ -44,6 +44,16 @@
 
 #define NOT_FOUND -1
 
+/*
+ * Return true if database is any of the three
+ * system databases "master", "tempdb", "msdb"
+ * else return false
+ */
+#define IS_BBF_SYSTEM_DB(dbname) \
```

Comment:
In `drop_bbf_db`, we have the same check.
`if ((strlen(dbname) == 6 && (strncmp(dbname, "master", 6) == 0)) ||
		((strlen(dbname) == 6 && strncmp(dbname, "tempdb", 6) == 0)) ||
		(strlen(dbname) == 4 && (strncmp(dbname, "msdb", 4) == 0)))` Let's replace that too.